workspaces/cli-scripts/src/config.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  createCliConfig,
+  Env,
+  CliConfigObject,
+  isEnvTrue,
+  PackageJson,
+} from '@useoptic/cli-shared';
+const pkg = require('../package.json') as PackageJson;
+
+const cliConfig = createCliConfig(pkg, process.env);
+
+export default {
+  ...cliConfig,
+
+  errors: {
+    sentry: cliConfig.env.production &&
```

Comment:
Is the production check redundant if we're injecting whether it is enabled? I would prefer the SENTRY__ENABLED to be the source of truth rather than SENTRY__ENABLED * env

Reply:
See [my original comment ](https://github.com/opticdev/optic/pull/450#discussion_r534184823)about why I decided against that:

> I've only ever found enabling sentry locally helpful in implementing the integration with sentry, and have found it super annoying to find development errors to be tracked because it was accidentally enabled, so I optimised with that hard-won experience in mind 😅.